### PR TITLE
Fixes for #415: Footer link change

### PR DESCRIPTION
**Pull request by cakleimeier**
- Remove "Legal" and "Press" links from footer
- Add "Success Stories" link to footer
- Change "Blog" link in footer
- Increase .footer-links-container width to 130px so "Success Stories" displays on 1 line

### CODE REVIEW

**Comment by KevinMulhern** [comment]
👍 LGTM

**Comment by arun1595** [comment]
@cakleimeier Good work. Thanks :)

File: app/views/shared/_footer.html.erb [hunk 1]
```diff
@@ -12,11 +12,10 @@
     <div class="footer-links-container">
       <%= link_to "About", about_path %>
       <%= link_to "FAQ", faq_path %>
-      <%= link_to "Blog", "http://1000hours.io" %>
+      <%= link_to "Blog", "https://medium.com/the-odin-project" %>
```
Comment (arun1595) [inline app/views/shared/_footer.html.erb]: Nitpick. Can you add a method `medium_blog_url` in `ApplicationHelper` and use that here?

```
def medium_blog_url
  'https://medium.com/the-odin-project'
end
```

Comment (cakleimeier) [inline app/views/shared/_footer.html.erb]: I'll give that a try. :)

Comment (arun1595) [inline app/views/shared/_footer.html.erb]: @cakleimeier Thanks!

**Pull request closed without merging**